From ee594b1a12833c06102de888248a361bc49cea09 Mon Sep 17 00:00:00 2001
From: Ovidiu Panait <ovidiu.panait@windriver.com>
Date: Fri, 18 Aug 2023 12:20:40 +0300
Subject: [PATCH] tests: add .broken files for 04update-uuid and
 07revert-inplace

04update-uuid and 07revert-inplace tests are unreliable and fail intermittently
on the autobuilder. Unfortunately, the failures cannot be reproduced locally
and the logs cannot be retrieved from the AB.

Mark the testcases as BROKEN to skip them when running ptest.

Upstream-Status: Inappropriate

Signed-off-by: Ovidiu Panait <ovidiu.panait@windriver.com>
---
 tests/04update-uuid.broken    | 1 +
 tests/07revert-inplace.broken | 1 +
 2 files changed, 2 insertions(+)
 create mode 100644 tests/04update-uuid.broken
 create mode 100644 tests/07revert-inplace.broken

diff --git a/tests/04update-uuid.broken b/tests/04update-uuid.broken
new file mode 100644
index 0000000..197b35b
--- /dev/null
+++ b/tests/04update-uuid.broken
@@ -0,0 +1 @@
+fails infrequently
diff --git a/tests/07revert-inplace.broken b/tests/07revert-inplace.broken
new file mode 100644
index 0000000..197b35b
--- /dev/null
+++ b/tests/07revert-inplace.broken
@@ -0,0 +1 @@
+fails infrequently
-- 
2.39.1

